### PR TITLE
Solved stress_test1_crash_during_exit_with_pr4888

### PR DESCRIPTION
**Pull request by s1-vivek**
Changed code such that deinit is called also when there is error in setup part of test case.

### CODE REVIEW

**Comment by sunghan-chang** [comment]
@s1-vivek @vibhor-m Please find our [commit rules](https://github.com/Samsung/TizenRT/wiki/Documentations#commit-rules) in the wiki. You should rewrite commit title and add description.

File: external/stress_tool/st_perf.c [hunk 1]
```diff
@@ -130,11 +130,15 @@ void _run_smoke(st_smoke *smoke)
...
 				// so remained procedures could be useless
 				// so stopping running smoke would be better
-				continue;
+				//continue;
+				printf("%s cnt= %d tc_%s_setup returned %d\n",unit->tc_name, cnt, unit->tc_name , ret);
 			}
 		}
 		st_elapsed_time duration;
 		ret = unit->tc(&duration);
+		if(ret!=STRESS_TC_PASS){
```
Comment (jeongchanKim) [inline external/stress_tool/st_perf.c:139]: Please add space after `if`.

**Review by jeongchanKim — COMMENTED** [review]
Please check the TizenRT Coding rule.

**Comment by vibhor-m** [comment]
> Please check the TizenRT Coding rule.

Suggestion will be taken care. Required changes will be done

**Comment by vibhor-m** [comment]
> @s1-vivek @vibhor-m Please find our [commit rules](https://github.com/Samsung/TizenRT/wiki/Documentations#commit-rules) in the wiki. You should rewrite commit title and add description.

Suggestion will be taken care and required code changes will be done

**Pull request closed without merging**